LoopKitUI/Views/QuantityScheduleEditor.swift

```diff
@@ -130,7 +130,7 @@ struct QuantityScheduleEditor<ActionAreaContent: View>: View {
     private var guardrailWarningIfNecessary: some View {
         let crossedThresholds = self.crossedThresholds
         return Group {
-            if !crossedThresholds.isEmpty {
+            if !crossedThresholds.isEmpty && (userDidTap || mode == .settings || mode == .legacySettings) {
```

Comment:
Hm...is this right?  we don't show the warning during acceptance flow?  (I am starting to get confused about which UI is for which place...)

Reply:
Yep, this is to ensure the instructional content isn't shown at the same time as a guardrail warning (the instructional content goes away when the user taps, so the warning will be shown eventually)